torchsde/_core/methods/milstein.py

```diff
@@ -12,30 +12,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ...settings import SDE_TYPES, NOISE_TYPES, LEVY_AREA_APPROXIMATIONS
+import math
+
+import torch
+
+from ...settings import SDE_TYPES, NOISE_TYPES, LEVY_AREA_APPROXIMATIONS, METHOD_OPTIONS as opt
```

Comment:
Is there a reason it's `as opt` rather than just using `METHOD_OPTIONS`?

Reply:
Not really - changed to full name.